app/models/user.rb

```diff
@@ -33,4 +31,8 @@ def unfollow(user)
     relationship = active_relationships.find_by(following_id: user.id)
     relationship&.destroy!
   end
+
+  def name_or_email
+    name.nil? ? email : name
```

Comment:
Railsの場合、こう書くこともできますー。

```suggestion
    name.presence || email
```

そもそも

```suggestion
    name || email
```

だけでもほぼ同じですけどね

Reply:
`||`で置き換えました！